systemtests/src/main/java/io/enmasse/systemtest/time/TimeMeasuringSystem.java

```diff
@@ -37,6 +37,8 @@
     private String testClass;
     private String testName;
 
+    private boolean logResults = true;
```

Comment:
Is it better enabled/disabled via Environment.java ?

Reply:
agree, but first need support to set that env var in our pipelines